gulpfile.js

```diff
@@ -1,5 +1,280 @@
+    //setBranchBaseUrl();
+  }
+
+  var hugo_args = [];
+  if (process.env.SITE_BASEURL) {
+    gutil.log(gutil.colors.cyan('build:website'), 'Using environment-specified BaseUrl: ' + process.env.SITE_BASEURL);
+    hugo_args = [ '-b', process.env.SITE_BASEURL];
+  }
+
+  var hugo = spawn('hugo', hugo_args);
+
+  hugo.stdout.on('data', function (data) {
+    gutil.log(gutil.colors.blue('build:website'), '\n' + data);
+  });
+
+  hugo.on('error', done);
+  hugo.on('close', done);
+
+});
+
+gulp.task('watch', function () {
+  gutil.log(gutil.colors.cyan('watch'), 'Watching assets for changes');
+  gulp.watch('./assets/styles/**/*.scss', [ 'styles:homepage' ]);
+  //gulp.watch('./assets/scripts/**/*.js', [ 'scripts' ]);
+  gulp.watch('./assets/images/**/*', [ 'images' ]);
+});
+
+gulp.task('website', [ 'build', 'watch' ], function (done) {
+
+  var buildDrafts = '--buildDrafts';
+
+  if (cFlags.production) {
+    buildDrafts = '';
+  }
+
+  var hugo = spawn('hugo', [ 'server', buildDrafts ]);
+
+  hugo.stdout.on('data', function (data) {
+    gutil.log(gutil.colors.blue('website'), '\n' + data);
+  });
+
+  hugo.on('error', done);
+  hugo.on('close', done);
+
+});
+
+gulp.task('default', function (done) {
+  printPackageInfo();
+  gutil.log('Available tasks');
+  gutil.log('$', gutil.colors.magenta('gulp watch'));
+  gutil.log('Watch for changes and build the asset-pipeline');
+  gutil.log('$', gutil.colors.magenta('gulp [ production, no-test ] build'));
+  gutil.log('Build the asset-pipeline with optional production and no-test flags');
+  gutil.log('$', gutil.colors.magenta('gulp clean-all'));
+  gutil.log('Removes files and directories generated by gulp');
+  gutil.log('$', gutil.colors.magenta('gulp [ production ] website'));
+  gutil.log('Runs the gulp watch and hugo serve');
+  gutil.log('$', gutil.colors.magenta('gulp [ production ] build:website'));
+  gutil.log('Build the asset-pipeline and the website using Hugo');
+  done();
+});
+
+function printPackageInfo () {
+  gutil.log(
+    gutil.colors.yellow('v' + pkg.version),
+    gutil.colors.green(pkg.name)
+  );
+  gutil.log();
+  gutil.log(gutil.colors.red(' ______ ______ _____    '));
+  gutil.log(gutil.colors.red('/\\  ___/\\  ___/\\  __-.  '));
+  gutil.log(gutil.colors.blue('\\ \\  __\\ \\  __\\ \\ \\/\\ \\ '));
+  gutil.log(gutil.colors.blue(' \\ \\_\\  \\ \\_\\  \\ \\____- '));
+  gutil.log(gutil.colors.white('  \\/_/   \\/_/   \\/____/ '));
+  gutil.log(gutil.colors.red(' __   ________  ______  ______  ______  ______  __   __  '));
+  gutil.log(gutil.colors.red('/\\ \\ / /\\  __ \\/\\__  _\\/\\  ___\\/\\  ___\\/\\  __ \\/\\ \\ / /  '));
+  gutil.log(gutil.colors.blue('\\ \\ \\\'/\\ \\ \\/\\ \\/_/\\ \\/\\ \\  __\\\\ \\ \\__ \\ \\ \\/\\ \\ \\ \\\'/   '));
+  gutil.log(gutil.colors.blue(' \\ \\__| \\ \\_____\\ \\ \\_\\ \\ \\_____\\ \\_____\\ \\_____\\ \\__|   '));
+  gutil.log(gutil.colors.white('  \\/_/   \\/_____/  \\/_/  \\/_____/\\/_____/\\/_____/\\/_/    '));
```

Comment:
🎉 

Thinking maybe just "FFD VOTE" would be cooler, but open to being convinced 😉